lib/rubocop/cop/method_complexity_patch.rb

```diff
@@ -0,0 +1,10 @@
+# Monkey patching! Here be dragons.
+module RuboCop
+  module Cop
+    module MethodComplexity
+      def add_offense(node, loc, message = nil, severity = nil)
```

Comment:
Well, I've added a comment now, but it's not short. I'm not sure how much briefer it can be & still be good english without just cutting it down to "see commit message for why this is begin done."